frappe/www/attribution.py

```diff
@@ -24,6 +25,7 @@ def get_context(context):
 	context.apps = apps
 
 
+@cache
```

Comment:
Storing all this app information in memory isn't needed at all - its a rarely visited page to begin with. We could use the few bytes per worker for something else.